client/src/components/NavBar/AuthBars/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,71 @@
+import Button from '@material-ui/core/Button';
+import { Box, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import Notification from './Notification/Notification';
+import { useState, useEffect } from 'react';
+import getUnread from '../../../../helpers/APICalls/getAllUnreadNotifications';
+
+interface notification {
+  type: string;
+  title: string;
+  description: string;
+  read: boolean;
+  date: Date;
+  image: string;
+}
+
+const Notifications = (): JSX.Element => {
+  const classes = useStyles();
+  const [notifications, setNotifications] = useState<notification[]>([
+    {
+      type: '',
+      title: 'filler',
+      description: '',
+      read: false,
+      date: new Date('1/1/2000'),
+      image: '',
+    },
+  ]);
+
+  useEffect(() => {
```

Comment:
As opposed to rewriting this code wouldn't it be better to store it in a context and share the state across the component here and above? :)